tekton/pipeline_run.go

```diff
@@ -50,7 +50,7 @@ const (
 	PipelineTypeRelease = "release"
 
 	// pipelinesAsCodeMetadataPrefix is the prefix for pipelines-as-code metadata
-	pipelinesAsCodeMetadataPrefix = "pipelinesascode.tekton.dev"
+	pipelinesAsCodeMetadataPrefix = "pac.test.appstudio.openshift.io"
```

Comment:
Why **test**?

Reply:
replying on slack

Reply:
Quoting from [hacbs-1385](https://issues.redhat.com/browse/HACBS-1385):

> Acceptance Criteria
>
>The integration controller replaces the prefix ("pipelinesascode.tekton.dev/") on the PaC labels with the AppStudio test prefix ("pac.test.appstudio.openshift.io/") when adding them to the Integration PipelineRun.

The **test** string is also reflected in the integration-service PR:
https://github.com/redhat-appstudio/integration-service/pull/69